### PR TITLE
Update references to Global Administrators post-flattening

### PR DESCRIPTION
**Pull request by AnnikaCodes**
85% of this was automated so if there are issues please let me know.

Also, translators are probably needed to update the translations - I changed the symbols (and obviously English) but more changes are probably necessary.

### CODE REVIEW

File: translations/french.json [hunk 1]
```diff
@@ -36,9 +35,9 @@
 		"DATA COMMANDS": "COMMANDES DES DONNÉES",
 		"DRIVER COMMANDS": "COMMANDES DES DRIVERS",
 		"MODERATOR COMMANDS": "COMMANDES DES MODERATEURS",
-		"LEADER COMMANDS": "COMMANDES DES LEADERS",
+		"ADMIN COMMANDS": "COMMANDES DES LEADERS",
```
Comment (sparkychildcharlie) [inline translations/french.json:38]: Commandes des administrateurs

File: translations/german.json [hunk 2]
```diff
@@ -36,9 +35,9 @@
 		"DATA COMMANDS": "DATENBEFEHLE",
 		"DRIVER COMMANDS": "BEFEHLE FÜR DRIVER",
 		"MODERATOR COMMANDS": "BEFEHLE FÜR MODERATOREN",
-		"LEADER COMMANDS": "BEFEHLE FÜR LEADER",
+		"ADMIN COMMANDS": "BEFEHLE FÜR LEADER",
```
Comment (sparkychildcharlie) [inline translations/german.json:38]: BEFEHLE FÜR ADMINISTRATOREN

File: translations/hindi.json [hunk 3]
```diff
@@ -40,9 +39,9 @@
 		"DATA COMMANDS": "डेटा संबंधी कमांड",
 		"DRIVER COMMANDS": "Drivers के कमांड",
 		"MODERATOR COMMANDS": "Moderators के कमांड",
-		"LEADER COMMANDS": "Leaders के कमांड",
+		"ADMIN COMMANDS": "Leaders के कमांड",
```
Comment (sparkychildcharlie) [inline translations/hindi.json:42]: (I've never seen hindi before, but from the pattern)
Administrators के कमांड

File: translations/dutch.json [hunk 4]
```diff
@@ -39,9 +38,9 @@
 		"DATA COMMANDS": "DATA COMMANDO'S",
 		"DRIVER COMMANDS": "DRIVER COMMANDO'S",
 		"MODERATOR COMMANDS": "MODERATOR COMMANDO'S",
-		"LEADER COMMANDS": "LEADER COMMANDO'S",
+		"ADMIN COMMANDS": "LEADER COMMANDO'S",
```
Comment (sparkychildcharlie) [inline translations/dutch.json:41]: ADMINISTRATOR COMMANDO'S

(line 21 uses administrator in the translation + the other ranks are also directly copied over from english)

File: translations/italian.json [hunk 5]
```diff
@@ -39,9 +38,9 @@
 		"DATA COMMANDS": "COMANDI PER I DATI",
 		"DRIVER COMMANDS": "COMANDI PER I DRIVER",
 		"MODERATOR COMMANDS": "COMANDI PER I MODERATORI",
-		"LEADER COMMANDS": "COMANDI PER I LEADER",
+		"ADMIN COMMANDS": "COMANDI PER I LEADER",
```
Comment (sparkychildcharlie) [inline translations/italian.json:41]: comandi per i amministratori

(i dont speak italian either but google translate gives this for "administrators", and it fits the pattern with "moderators", which is also a real word)

File: translations/japanese.json [hunk 6]
```diff
@@ -39,9 +38,9 @@
 		"DATA COMMANDS": "ポケモンデータコマンド",
 		"DRIVER COMMANDS": "ドライバー（％）コマンド",
 		"MODERATOR COMMANDS": "モデレーター（＠）コマンド",
-		"LEADER COMMANDS": "リーダー（＆）コマンド",
+		"ADMIN COMMANDS": "リーダー（＆）コマンド",
```
Comment (sparkychildcharlie) [inline translations/japanese.json:41]: アドミニストレーター（＆）コマンド

File: translations/portuguese.json [hunk 7]
```diff
@@ -39,9 +38,9 @@
 		"DATA COMMANDS": "COMANDOS DE DADOS",
 		"DRIVER COMMANDS": "COMANDOS DE DRIVER",
 		"MODERATOR COMMANDS": "COMANDOS DE MODERATOR",
-		"LEADER COMMANDS": "COMANDOS DE LEADER",
+		"ADMIN COMMANDS": "COMANDOS DE LEADER",
```
Comment (sparkychildcharlie) [inline translations/portuguese.json:41]: COMANDOS DE ADMINISTRATOR
(direct copy from the english just like the other ranks)

File: translations/simplifiedchinese.json [hunk 8]
```diff
@@ -39,9 +38,9 @@
 		"DATA COMMANDS": "数据指令",
 		"DRIVER COMMANDS": "见习管理指令",
 		"MODERATOR COMMANDS": "管理员指令",
-		"LEADER COMMANDS": "领袖指令",
+		"ADMIN COMMANDS": "领袖指令",
```
Comment (sparkychildcharlie) [inline translations/simplifiedchinese.json:41]: 总管指令

(copied from line 21)

File: translations/spanish.json [hunk 9]
```diff
@@ -39,9 +38,9 @@
 		"DATA COMMANDS": "COMANDOS DE DATOS",
 		"DRIVER COMMANDS": "COMANDOS DE DRIVER",
 		"MODERATOR COMMANDS": "COMANDOS DE MODERATOR",
-		"LEADER COMMANDS": "COMANDOS DE LEADER",
+		"ADMIN COMMANDS": "COMANDOS DE LEADER",
```
Comment (sparkychildcharlie) [inline translations/spanish.json:41]: COMANDOS DE ADMINISTRATOR

File: translations/traditionalchinese.json [hunk 10]
```diff
@@ -39,9 +38,9 @@
 		"DATA COMMANDS": "數據指令",
 		"DRIVER COMMANDS": "見習管理指令",
 		"MODERATOR COMMANDS": "管理員指令",
-		"LEADER COMMANDS": "領袖指令",
+		"ADMIN COMMANDS": "領袖指令",
```
Comment (sparkychildcharlie) [inline translations/traditionalchinese.json:41]: 總管指令

**Comment by sparkychildcharlie** [comment]
Also hate to say it, but if I personally would love to make it so that there's something like ``Chat.requiresAuth('mute')`` which automatically generates the line depending on the ranks the servers have set up.  Yes, main would have admins ripped (seriously ~ looks better on Zarel than &), but side servers who already exist won't have this.

Would love to have support provided for side servers as well.  I don't mind doing this myself in a future pull request.

File: translations/italian.json [hunk 11]
```diff
@@ -38,7 +38,7 @@
 		"DATA COMMANDS": "COMANDI PER I DATI",
 		"DRIVER COMMANDS": "COMANDI PER I DRIVER",
 		"MODERATOR COMMANDS": "COMANDI PER I MODERATORI",
-		"ADMIN COMMANDS": "COMANDI PER I LEADER",
+		"ADMIN COMMANDS": "COMANDI PER I AMMINISTARTORI",
```
Comment (AnnikaCodes) [inline translations/italian.json:41]:
```suggestion
		"ADMIN COMMANDS": "COMANDI PER I AMMINISTRATORI",
```
typo zzz

File: translations/dutch.json (hunk 4 above)
Comment (DeltaCoderr) [inline translations/dutch.json]: The "LEADER COMMANDO'S" should be "ADMIN COMMANDO'S"?

Comment (AnnikaCodes) [reply translations/dutch.json]: Yes — this pull request removes the line ``` "LEADER COMMANDS": "LEADER COMMANDO'S", ```, replacing it with ``` "ADMIN COMMANDS": "ADMINISTRATOR COMMANDO'S", ```. Perhaps you were looking at outdated code?

Comment (DeltaCoderr) [reply translations/dutch.json]: Ah, my apologies

**Comment by Zarel** [comment]
I'm not sure if this is complete yet, but it looks like a substantial marginal improvement so I'll accept it now. In case this isn't everything, obviously feel free to submit more fixes in future pull requests.

**Pull request merged**